compiler/frontend/src/string_to_rcst/expression.rs

```diff
@@ -405,6 +405,31 @@ fn match_case(input: &str, indentation: usize) -> Option<(&str, Rcst)> {
     let (input, whitespace) = whitespaces_and_newlines(input, indentation, true);
     let pattern = pattern.wrap_in_whitespace(whitespace);
 
+    let (input, condition) = if let Some((input, condition_comma)) = comma(input) {
+        let (input, whitespace) = whitespaces_and_newlines(input, indentation, true);
+        let condition_comma = condition_comma.wrap_in_whitespace(whitespace);
+        if let Some((input, condition_expresion)) = expression(
```

Comment:
Small typo: `condition_expresion` is missing one “s”

Reply:
If you completely addressed a review comment and didn't write an answer, you can directly resolve the conversation yourself